phpunit-integration.xml

```diff
@@ -9,6 +9,7 @@
     </exclude>
     <report>
       <clover outputFile="coverage-integration.xml" />
+      <html outputDirectory="reports/integration-coverage"/>
```

Comment:
Is there a way to get combined coverage figures for the two test types? It would be great to have numbers that are comparable to the joint summary Codecov gives.

Reply:
That's what I was tying to do with codecov, but I ran into problem and gave up. Probably we could have PHPUnit config that runs all the tests at once to get combined coverage though.